### PR TITLE
Cico juniper

### PR DESCRIPTION
**Pull request by mtraneis**

### CODE REVIEW

File: workflows-scripts/circulation/check-in-check-out/circulation_checkInCheckOut.jmx [hunk 1]
```diff
@@ -15,7 +15,7 @@
         <collectionProp name="Arguments.arguments">
           <elementProp name="HOSTNAME" elementType="Argument">
             <stringProp name="Argument.name">HOSTNAME</stringProp>
-            <stringProp name="Argument.value">${__P(HOSTNAME,)}</stringProp>
+            <stringProp name="Argument.value">${__P(HOSTNAME,[host-name])}</stringProp>
```
Comment (javalkar-varun) [inline workflows-scripts/circulation/check-in-check-out/circulation_checkInCheckOut.jmx:18]: `HOSTNAME` is already stored above. Do we still need `[hostname]`?

**Comment by javalkar-varun** [comment]
If this change is specific to Juniper then maybe we should wait until we are done testing with Iris release? So, if we run CICO test for Iris env using these changes, will it work?

**Review by javalkar-varun — CHANGES_REQUESTED** [review]

**Comment by mtraneis** [comment]
> If this change is specific to Juniper then maybe we should wait until we are done testing with Iris release? So, if we run CICO test for Iris env using these changes, will it work?

Yes, good point. Do not need to merge yet.

File: workflows-scripts/circulation/check-in-check-out/circulation_checkInCheckOut.jmx (hunk 1 above)
Comment (mtraneis) [reply workflows-scripts/circulation/check-in-check-out/circulation_checkInCheckOut.jmx:18]: It's just a placeholder indicating the real value. We once had something similar to it but I removed it by accident.

**Review by javalkar-varun — APPROVED** [review]

**Pull request merged**